Description: Correct the output of --help falsely pointing to wrong long options.
Origin: other
Author: Didier Raboud <didier@raboud.com>
Bug: http://bugs.debian.org/535445
Last-Update: 2009-07-02 
Index: usb-modeswitch.git/usb_modeswitch.c
===================================================================
--- usb-modeswitch.git.orig/usb_modeswitch.c	2009-07-02 11:16:39.000000000 +0200
+++ usb-modeswitch.git/usb_modeswitch.c	2009-07-02 11:17:01.000000000 +0200
@@ -249,8 +249,8 @@
 				printf (" -r, --response-endpoint [nr]  where from read the response (optional)\n");
 				printf (" -d, --detach-only             just detach the storage driver\n");
 				printf (" -H, --huawei-mode             apply a special procedure\n");
-				printf (" -S, --sierraMode              apply a special procedure\n");
-				printf (" -O, --sonyMode                apply a special procedure\n");
+				printf (" -S, --sierra-mode             apply a special procedure\n");
+				printf (" -O, --sony-mode               apply a special procedure\n");
 				printf (" -R, --resetUSB                reset the device in the end\n");
 				printf (" -c, --config [filename]       load different config file\n");
 				printf (" -Q, --quiet                   don't show progress or error messages\n");
